styles/global.scss

```diff
@@ -294,3 +297,72 @@ hr {
     border-top: 1px solid theme('border-color');
   }
 }
+.border {
+  @include ktheme() {
+    border: 1px solid theme('border-color')
+  }
+}
+
+
+//colors
```

Comment:
have you seen in `styles/abstracts/variables.scss` it should do the work you looking for

Reply:
honestly i could not figure it out